src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/import.py

```diff
@@ -35,40 +33,36 @@ def acr_import(cmd,
     _, resource_group_name = validate_managed_registry(
         cmd, registry_name, resource_group_name, IMPORT_NOT_SUPPORTED)
 
+    ImportImageParameters, ImportSource, ImportMode = cmd.get_models(
+        'ImportImageParameters', 'ImportSource', 'ImportMode')
+
     if not source:
         raise CLIError(INVALID_SOURCE_IMAGE)
-    source_image = source
 
-    slash = source.find('/')
+    import re
+    ACR = re.match(r'[0-9a-zA-Z/]+(\.azurecr\.)(io|cn|us|de)', source)
```

Comment:
What if the image is called `abc.azurecr.io`?

Reply:
It does not matter whether the image is "abc.azurecr.io" or not. The match here is to ensure that we do not leave out possible login server at the first hand if it ever was. However, if it is not, it will do no harm to what is coming after.

Reply:
```
docker tag hello-world youregistry.azurecr.io/abc.azurecr.io:v1
docker push youregistry.azurecr.io/abc.azurecr.io:v1
az acr import --source abc.azurecr.io:v1 -r yourregistry
```

Reply:
Besides you are parsing by `/` anyway, why wasting time to have this if condition.

Reply:
Matching with regular expression is better over searching for slash because repository name itself could contain slashes.